components/button/floating-buttons.js

```diff
@@ -0,0 +1,207 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
```

Comment:
I think they should include colors. I somewhat just ran into this as an issue where when I was trying to get the visual diff html page setup, the button colors weren't showing up, and it turned out that it would need to be imported into that page. This hadn't been seen previously as with subtle button and icon button, icons was imported which imported colors. So overall it just seems safest to me to import it where we know it's needed just in case.